amills12/SignLanguageGame
# Me Controller Testing

Rotated leap motion recognition and other changes can be seen in DL Sign Hero.

Testing:
- Make sure the hands are in the correct position relative to the LMC and in the game
- Make sure the recognition is accurate for numbers 1-9
